test/CompletionExtSample/CompletionExt1.cs

```diff
@@ -0,0 +1,88 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Linq;
+using Microsoft.SqlTools.ServiceLayer.Connection;
+using Microsoft.SqlTools.ServiceLayer.LanguageServices.Completion;
+using Microsoft.SqlTools.ServiceLayer.LanguageServices.Completion.Extension;
+using Microsoft.SqlTools.ServiceLayer.LanguageServices.Contracts;
+
+namespace Microsoft.SqlTools.Test.CompletionExtension
+{
+    [Export(typeof(ICompletionExtensionProvider))]
+    public class CompletionExtProvider1 : ICompletionExtensionProvider
```

Comment:
what is the point of the `1` suffix.  are we going to be adding a `2` set of classes?

Reply:
We could add more extensions for testing purpose.

Reply:
hmm, ok, seems unusual to number classes when there is only one of them.

Reply:
Removed the numbering.